pkg/parquetconverter/metrics.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package parquetconverter
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type parquetConverterMetrics struct {
+	blocksConverted       prometheus.Counter
+	blocksConvertedFailed prometheus.Counter
+	conversionDuration    prometheus.Histogram
+	tenantsDiscovered     prometheus.Gauge
+}
+
+func newParquetConverterMetrics(reg prometheus.Registerer) parquetConverterMetrics {
+	var m parquetConverterMetrics
+
+	m.blocksConverted = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "cortex_parquet_converter_blocks_converted_total",
+		Help: "Total number of blocks converted to parquet format",
+	})
+
+	m.blocksConvertedFailed = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "cortex_parquet_converter_blocks_converted_failed_total",
+		Help: "Total number of blocks that failed to convert to parquet format",
+	})
+
+	m.conversionDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+		Name: "cortex_parquet_converter_conversion_duration_seconds",
+		Help: "Time taken to convert a block to parquet format",
+		// Buckets
+		// 1 0.008s  (8ms)
+		// 2 0.032s  (32ms)
+		// 3 0.128s  (128ms)
+		// 4 0.512s  (512ms)
+		// 5 2.048s  (2048ms)
+		// 6 8.192s  (8192ms)
+		// 7 32.768s (32768ms)
+		// 8 +Inf
```

Comment:
Do we think this is a good spread to capture the block conversion times? The conversions for larger blocks took a lot longer than 32s on my machine with 16 cores working on it

Reply:
Ha! you are right, was thinking about adjusting later. Going to try with:
```
❯ go run main.go  0.500 2 10
// Buckets
// 1 0.500s  (500ms)
// 2 1.000s  (1000ms)
// 3 2.000s  (2000ms)
// 4 4.000s  (4000ms)
// 5 8.000s  (8000ms)
// 6 16.000s  (16000ms)
// 7 32.000s  (32000ms)
// 8 64.000s  (64000ms)
// 9 128.000s  (128000ms)
// 10 256.000s  (256000ms)
// 11 +Inf
```